common/component/kafka/metadata.go

```diff
@@ -38,7 +38,6 @@ const (
 	certificateAuthType  = "certificate"
 	clientCert           = "clientCert"
 	clientKey            = "clientKey"
-	consumeRetryEnabled  = "consumeRetryEnabled"
```

Comment:
Why remove this?

Reply:
> Why remove this?

This is an unused constant, so I removed it.